modules/cnv-add-nic-to-vm.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template
+//
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:cnv-vm-add-nic:
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:cnv-vm-add-nic-template:
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-add-nic_{context}"]
+
+= Adding a network interface to a {object}
+
+.Procedure
+
+. From the *{object-gui}* tab, select the {object} you created.
+. Select the *Network Interfaces* tab.
+. Click *Add NIC*.
+. In the *Add NIC* window, specify the *Name*, *Model*, *Network*, *Type*,
+and *MAC Address* of the network interface.
+. For the virtual machine to be accessible, it must be restarted. Click *Add NIC* to add the network interface without restarting the virtual
+machine.
+. Use the drop-down lists and check boxes to edit the network
+interface configuration. When you edit an existing network interface, do
+one of the following:
+.. Click *Apply and Restart VM* to add the network interface and restart
+the virtual machine.
+.. Click *Save Changes* to add the network interface without restarting the
+virtual machine.
```

Comment:
Are both of these sub-steps required or are these options? 
If options, which should I pick.

Reply:
Related to your previous comment @mburke5678 --- the "Apply and Restart" button is not available in this release, so the step now reads only "Click *Save Changes*" as we have previously clarified the VM has to be restarted manually.